docs/resources/tozny_realm_group_with_access_policy.md

```diff
@@ -0,0 +1,143 @@
+  client_registration_token = tozny_client_registration_token.realm_registration_token.token
+  realm_name                = tozny_realm.my_organizations_realm.realm_name
+  name                      = "broker${tozny_realm.my_organizations_realm.realm_name}"
+  persist_credentials_to    = "terraform"
+}
+
+# A resource for delegating authority to another Tozny client
+# (either hosted by Tozny or some other third party) to broker realm
+# activities (such as email recovery).
+resource "tozny_realm_broker_delegation" "allow_tozny_hosted_brokering_policy" {
+  # Block on and use client credentials generated from the provisioned account
+  depends_on = [
+    tozny_account.autogenerated_tozny_account,
+    tozny_realm_broker_identity.broker_identity,
+  ]
+  client_credentials_config         = tozny_account.autogenerated_tozny_account.config
+  realm_broker_identity_credentials = tozny_realm_broker_identity.broker_identity.credentials
+  use_tozny_hosted_broker           = true
+}
+
+resource "tozny_realm_role" "mpc_approval_role" {
+  depends_on = [
+    tozny_realm.my_organizations_realm,
+  ]
+  client_credentials_config = tozny_account.autogenerated_tozny_account.config
+  realm_name                = tozny_realm.my_organizations_realm.realm_name
+  name                      = "approving_role"
+  description               = "a role"
+}
+
+resource "tozny_realm_group" "group_with_mpc" {
+  depends_on = [
+    tozny_realm.my_organizations_realm,
+    tozny_realm_role.mpc_approval_role
+  ]
+  client_credentials_config = tozny_account.autogenerated_tozny_account.config
+  name                      = "MPC Group"
+  realm_name                = tozny_realm.my_organizations_realm.realm_name
+  // Only one access policy can be attached to a group at this time
+  access_policies {
+    approval_role_ids               = [tozny_realm_role.mpc_approval_role.realm_role_id]
+    required_approvals              = 3
+    maximum_access_duration_seconds = 1600
+    plugin_type                     = "jira" // Tozny currently only supports "jira"
+    plugin_id                       = "pluginID"
+    plugin_mpc_flow_source          = "pluginSource"
+  }
+}
+```
+
+## Argument Reference
+
+### Top-level Arguments
+
+- `client_credentials_filepath` - (Optional) The filepath to Tozny client credentials for the Terraform provider to use when provisioning this group. Omit if using `client_credentials_config`.
+- `client_credentials_config` - (Optional) A JSON string containing Tozny client credentials for the provider to use when provisioning this group. Omit if using `client_credentials_filepath`.
+- `group_id` - (Computed) Service defined unique identifier for the group.
+- `realm_name` - (Required) The name of the realm with which to associate the group.
+- `name` - (Required) User defined name for the group.
+- `access_policies` - (Optional) The list of access policies to attach to the group.
```

Comment:
need to also document the schema for this, example here
https://github.com/tozny/terraform-provider-tozny/blob/master/docs/resources/tozny_realm_application.md#oidc-settings-schema